plugins/package-managers/node/src/main/kotlin/PackageJson.kt

```diff
@@ -80,6 +80,11 @@ private fun transformPackageJson(element: JsonElement): JsonElement {
     // Readd licenses as plain primitives.
     content["licenses"] = JsonArray(licenses.map { JsonPrimitive(it) })
 
+    // Remove "*dependencies" nodes which are empty array nodes.
```

Comment:
Maybe be clear that this is invalid syntax, and say "... invalid nodes which are empty array nodes instead of objects"?